static/app/components/confirm.tsx

```diff
@@ -315,6 +320,7 @@ class ConfirmModal extends React.Component<ModalProps, ModalState> {
                 priority={priority}
                 onClick={this.handleConfirm}
                 autoFocus
+                aria-label={typeof confirmText === 'string' ? confirmText : t('Confirm')}
```

Comment:
so it's ok to have `aria-label` and `children` with the same text?

When writing tests, should we give preference to aria-labels or texts?

Reply:
> so it's ok to have aria-label and children with the same text?

It's not wrong, just unnecessary. I'm doing it here to assure typescript that at least some aria-label will be present.

> When writing tests, should we give preference to aria-labels or texts?

It heavily depends on the situation:
https://develop.sentry.dev/frontend/using-rtl/#querying